_data/projects/nunit.yml

```diff
@@ -2,11 +2,18 @@ name: NUnit
 desc: NUnit is the most popular and widely used unit test framework for .NET
 site: https://github.com/nunit/nunit
 tags:
+- NUnit
 - ".NET"
+- ".NET Core"
+- dotnet
+- dotnetcore
```

Comment:
`dotnet` and `dotnetcore` are a little redundant here, would you mind removing them? The `.NET` and `.NET Core` tags are already used by other projects.